### PR TITLE
Improve OpenGraph metadata

### PR DESCRIPTION
**Pull request by Rygel42**
- URL was kittyinudao.eth.link instead of .limo
- replaced the image with the Kitty full logo instead of the ooutline logo

### CODE REVIEW

File: src/config.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import Head from 'next/head'
 import Script from 'next/script'
 
 const titleDefault = 'Kitty Inu DAO'
-const url = 'https://kittyinudao.eth.link/'
+const url = 'https://kittyinudao.eth.limo'
```
Comment (axtina) [inline src/config.jsx:5]: should be `https://kitty-inu-dao.eth.limo/`

**Review by axtina — CHANGES_REQUESTED** [review]
update the url to proper one for the website - current one is the one that is used for snapshot voting set up not the website.

File: src/config.jsx (hunk 1 above)
Comment (Rygel42) [reply src/config.jsx]: Done.

**Review by axtina — APPROVED** [review]
looks good now

**Pull request merged**